tests/test_upstream_issue.py

```diff
@@ -212,6 +212,40 @@ def test_github_issues_no_token(self,
         self.mock_github_repo.get_issue.assert_not_called()
         self.mock_github_issue.get_comments.assert_not_called()
 
+    @mock.patch('sync2jira.intermediary.Issue.from_github')
+    @mock.patch(PATH + 'Github')
+    @mock.patch(PATH + 'get_all_github_data')
+    def test_github_issues_no_token(self,
```

Comment:
I think this function name duplicates (and replaces??) the function defined at line 162, so it should be renamed here.

Reply:
Oops, forgot to change it!